templates/contacts/contact_list.haml

```diff
@@ -511,3 +511,15 @@
       modal.show();
     });
     {% endif %}
+
+    {% if error_message %}
```

Comment:
If you have the Django messages function like this `messages.error(self.request, 'Here is the message')` in your mixin, you would not need of this block of code